### PR TITLE
Ft landing page 171973226

### PR DESCRIPTION
**Pull request by makwasi1**
#### what does the PR do?
This allows a user to register his details.

#### Description of the task to be completed?

- create a landing page with a registration form.
- A button that allows a new user to join
- A link to the login page for previous users.
- Form validation using javascript.
-storing user input into an array

### How can this be manually tested?
loading the page with gh-pages
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/171985101

### CODE REVIEW

File: index.htm [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </form>
+    </div>
+    </section>
+    <div class="footer">
+        
+        <p>    EduLibrary   </p>
+        
+    </div>
+</body>
+</html>
```
Comment (jennizalwango) [inline index.htm:62]: Good work @makwasi1  Please add a newline to this file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        <div id="values"></div>
+    </div>
+    </section>
+    <div class="footer">
+        
+        <p>    EduLibrary   </p>
+        
+    </div>
+</body>
+</html>
```
Comment (jennizalwango) [inline index.html]: @makwasi1 Please add a newline to this file

File: style.css [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    .home-main-section{
+        justify-content: center;
+    }
+    input[type=submit]{
+        
+        
+    }
+}
+
+
```
Comment (jennizalwango) [inline style.css:215]: Remove the rest of the line that is not being used.

**Review by jennizalwango — CHANGES_REQUESTED** [review]
Good work @makwasi1 Kindl attend to the requested changes then we will be good to go

File: app.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+	}
+regEx();
+
+}
+}
+
+
+    const data = new Array();
+    //const data = [];
+
```
Comment (jennizalwango) [inline app.js:57]: l don't think this line is not necessary

File: index.htm (hunk 1 above)
Comment (jennizalwango) [reply index.htm:62]: @makwasi1 attend to this too

File: index.html (hunk 2 above)
Comment (jennizalwango) [reply index.html]: @makwasi1 attend to this too

**Review by jennizalwango — CHANGES_REQUESTED** [review]
Please attend to the comments below

File: app.js (hunk 4 above)
Comment (makwasi1) [reply app.js:57]: but i am creating an empty array to store the data

**Pull request merged**